DigitalLearningSolutions.Web/Styles/supervisor/staffMemberCard.scss

```diff
@@ -0,0 +1,6 @@
+﻿$card-text-colour: #000000;
```

Comment:
Is there not a standard gds black colour for this in the framework?

Reply:
This is addressed.